packages/botonic-plugin-google-analytics/README.md

```diff
@@ -0,0 +1,49 @@
+# Botonic Plugin Google Analytics
+
+## What Does This Plugin Do?
+
+With this plugin you can track the user interaction with the bot or the bot's behaviour sending the information to [Google Analytics](https://analytics.google.com/). 
+
+## Setup
+
+1. Run `npm install --save @botonic/plugin-google-analytics` to install the plugin.
+2. Add it to the `src/plugins.js` file:
+
+**src/plugins.js**
+```javascript
+export const plugins = [
+  {
+    id: 'google_analytics',
+    resolve: require('@botonic/plugin-google-analytics'),
+    options: {
+      botName: 'YOUR_BOT_NAME',
+      trackingId: 'YOUR_TRACKING_ID'
+    }
+  }
+]
+```
+
+## Use
+
+The tracking needs to be done manually, enabling this plugin doesn't track any user interaction or bot's behaviour by default.
+You can do the tracking inside the `botonicInit` method, `render` method or even inside the [Webchat listeners](https://docs.botonic.io/concepts/webchat#webchat-listeners)
+
+```javascript
+static async botonicInit({ input, session, params, lastRoutePath, plugins }) {
+    const ga = plugins.google_analytics
+    
+    const eventName = 'botEvent'
+    const eventData = {
+      category: 'chatbot',
```

Comment:
please document what eventData/eventFields can/should contain:
* only the fields in the example or any fields?
* only strings and numbers, also nested objects, also class instances?

Reply:
@dpinol is it possible to add an `index.d.ts` for this project with definitions, or it has to be strictly a ts project?

Reply:
@dpinol @vanbasten17 I enhanced the documentation and I added descriptions of the eventFields argument.

Reply:
looking nice! 👌

Reply:
@vanbasten17 good point, we have index.d.ts for core and react, right? Can you help Sastre we this?

Reply:
@dpinol great! I'll have a look at it with @vanbasten17 in next PR